codalab/apps/web/templates/web/my/datasets.html

```diff
@@ -50,15 +51,21 @@
                                             <td>{{ dataset.key }}</td>
                                             <td>{{ dataset.description }}</td>
                                             <td>{{ dataset.type }}</td>
+                                            <td>{{ dataset.is_public|default:"" }}</td>
```

Comment:
Does this correctly turn a bool into a string? I think we had to do something with `yesno` to make this work right on v2. Something like:
```
{{dataset.is_public|yesno:"True,False"}}
```

Reply:
the existing code prints `"True"` or nothing if false

Reply:
Cool. Maybe I'm misremembering the issue then. 👍